Microsoft.Research/RegressionTest/ClousotTests/v.cs

```diff
@@ -48,6 +50,1516 @@ public TestContext TestContext
       }
     }
 
+    private static readonly Options[] TestOptions =
+    {
+        new Options(
```

Comment:
:memo: Each `new Options` here is drawn from an XML element from the original test inputs file.